src/odemis/driver/xt_client.py

```diff
@@ -21,9 +21,14 @@
 """
 import logging
 import math
+import notify2
```

Comment:
It will need to have `notification-daemon` installed, right? If so, you need to add it to the dependencies of Odemis, in `debian/control`.
I'm actually wondering if that extra package is really necessary... Especially this component is not supported in the latest version of Gnome (though we still have time to change as Ubuntu 22.04 still has it).

Reply:
@pieleric because xt_client.py is run with `sudo`, we need `notification-daemon` (i.e. a separate service which makes use of notification-daemon) for `notify2` to work. Should I add it to `debian/control`? or this [PR](https://bitbucket.org/delmic/ubuntu-delmic/pull-requests/5) will suffice?

Reply:
Let's add it to debian/control. We'll deal with it differently once it's gone...